crates/bevy_render/src/color/mod.rs

```diff
@@ -368,31 +368,31 @@ impl Color {
         )
     }
 
-    /// Get red in sRGB colorspace.
+    /// Get red
     pub fn r(&self) -> f32 {
         match self.as_rgba() {
             Color::Rgba { red, .. } => red,
             _ => unreachable!(),
         }
     }
 
-    /// Get green in sRGB colorspace.
+    /// Get green
     pub fn g(&self) -> f32 {
         match self.as_rgba() {
             Color::Rgba { green, .. } => green,
             _ => unreachable!(),
         }
     }
 
-    /// Get blue in sRGB colorspace.
+    /// Get blue
     pub fn b(&self) -> f32 {
         match self.as_rgba() {
             Color::Rgba { blue, .. } => blue,
             _ => unreachable!(),
         }
     }
 
-    /// Set red in sRGB colorspace.
+    /// Set red
```

Comment:
```suggestion
    /// Converts a Color to variant Color::Rgba and set red.
```

I think those should mention that the color will be converted to a different variant. Same for he `with_*` methods.

Reply:
Valid point. Already fixed the comments.